strax/config.py

```diff
@@ -203,6 +218,224 @@ def validate(self, config,
             config[self.name] = self.get_default(run_id, run_defaults)
 
 
+#Backward compatibility
+@export
+class Config(Option):
+    def __init__(self, **kwargs):
+        if 'name' not in kwargs:
+            kwargs['name'] = ''
+        super().__init__(**kwargs)
+
+    def __set_name__(self, owner, name):
+        self.name = name
+        takes_config = {name: self}
```

Comment:
Not sure i understand what you mean here, is the issue the readability or the functionality. this code performs the tasks that the takes_config decorator would perform on class instantiation. Since I dont use the decorator I have no need for it in this namespace and it seamed like the most descriptive name for what it holds, is it more confusing than descriptive?

Reply:
For me it was just readability